uaclient/api/tests/test_magic_attach_wait.py

```diff
@@ -0,0 +1,48 @@
+import mock
```

Comment:
Can we rename this file to `test_u_pro_attach_magic_wait_v1.py` or `test.u.pro.attach.magic.wait.v1.py`